packages/main/src/plugin/dockerode/libpod-dockerode.spec.ts

```diff
@@ -0,0 +1,57 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { afterEach, beforeAll, beforeEach, test } from 'vitest';
+import { type LibPod, LibpodDockerode } from '/@/plugin/dockerode/libpod-dockerode.js';
+import Dockerode from 'dockerode';
+import { rest } from 'msw';
+import { type SetupServer, setupServer } from 'msw/node';
```

Comment:
we already have `nock` library so I'm unsure we should add `msw` (or we switch from nock to msw everywhere)